README.rst

```diff
@@ -1,10 +1,27 @@
 XBlock to embed Instruqt track in Open edX
 ##########################################
 
-How to use
-**********
+About Instruqt
+**************
 
-This XBlock can be install using pip:
+Instruqt is a hands-on virtual IT Lab Platform for interactive learning. Virtual IT labs provide learners with hands-on sandbox environments.
+In which they can learn about products and technology.
```

Comment:
The sentence structure for this specific sentence is not correct. 
"Learners can use the platform to learn about products and technology".